quickstart/iOS/ZUMOAPPNAME/ZUMOAPPNAME/QSTodoListViewController.m

```diff
@@ -87,33 +113,28 @@ - (void) refresh
 #pragma mark * UITableView methods
 
 
-- (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle
- forRowAtIndexPath:(NSIndexPath *)indexPath
+- (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath *)indexPath
 {
     // Find item that was commited for editing (completed)
-    NSDictionary *item = [self.todoService.items objectAtIndex:indexPath.row];
+    NSManagedObject *item = [self.fetchedResultsController objectAtIndexPath:indexPath];
+    // TODO: map values to what we expect in SDK
+    NSDictionary *dict = @{ MSSystemColumnId:[item valueForKey:@"id"], MSSystemColumnVersion:[item valueForKey:@"ms_version"] };
```

Comment:
use the helper you asked for please: https://github.com/Azure/azure-mobile-services/blob/offline/sdk/iOS/src/MSCoreDataStore.h#L34